outputs.tf

```
@@ -22,3 +22,8 @@ output "api_oauth2_permission_scope_ids" {
   description = "A map of Oauth2 permission scope IDs."
   value       = azuread_application.this.oauth2_permission_scope_ids
 }
+
+output "app_role_ids" {
+  description = "A map of app role IDs."
```

Put the description as it is in the Terraform Registry docs? 

```suggestion
  description = "A mapping of app role values to app role IDs, intended to be useful when referencing app roles in other resources in your configuration."
```